src/tradingtimes/tradingTimes.es6

```diff
@@ -172,29 +173,40 @@ const initTradingWin = ($html) => {
               const sub_select = $('<select />');
               sub_select.appendTo(subheader);
               submarket_names = windows.makeSelectmenu(sub_select, {
-                list: result.submarket_names[market_names.val()],
+                list: Object.keys(header[market_names.val()]),
                 inx: 0,
                 changed: changedFn,
               });
               submarket_names.off('selectmenuchange', changed);
               submarket_names.on('selectmenuchange', changed);
             } else {
-            submarket_names.update_list(result.submarket_names[market_names.val()]);
+            submarket_names.update_list(Object.keys(header[market_names.val()]));
             submarket_names.off('selectmenuchange', changed);
             submarket_names.on('selectmenuchange', changed);
         }
 
           updateTable(result, market_names.val(), submarket_names.val());
-          processing_msg.hide();
-         
       };
 
-      const getCachedData = () => liveapi.cached.send({ trading_times: yyyy_mm_dd })
-      .then(result => refresh(result))
-      .catch((error) => {
-        $.growl.error({ message: error.message });
-        refresh({});
-      });
+      const getCachedData = () => {
+         const active_symbols_request = { active_symbols: 'brief' };
+         const asset_index_request = { trading_times: yyyy_mm_dd };
+         const processing_msg = $('#' + table.attr('id') + '_processing').show();
```

Comment:
can we rename this as `$processing_msg` and call `.show()` on the next line to make it more readable